card/spec/models/card/create_spec.rb

```diff
@@ -13,14 +13,14 @@
 describe Card, "created by Card.new " do
   before(:each) do
     Card::Auth.as_bot do
-      @c = Card.new :name=>"New Card", :content=>"Great Content"
+      @c = Card.new name: "New Card", content: "Great Content"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.